src/main/webapp/web.xml

```diff
@@ -11,6 +11,106 @@
 
     <servlet-mapping>
         <servlet-name>index</servlet-name>
-        <url-pattern>/index</url-pattern>
+        <url-pattern>/</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>createManufacturer</servlet-name>
+        <servlet-class>internet.shop.controllers.manufacturers.AddManufacturerController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>createManufacturer</servlet-name>
+        <url-pattern>/manufacturer/create</url-pattern>
```

Comment:
```suggestion
        <url-pattern>/manufacturers/create</url-pattern>
```
it is good practice to use plurals in URLs

Reply:
done